notebook/tests/test_notebookapp.py

```diff
@@ -139,7 +137,7 @@ def test_notebook_password():
             assert nb.password != ''
             passwd_check(nb.password, password)
 
-class TestingStopApp(notebookapp.NbserverStopApp):
+class StopTestApp(notebookapp.NbserverStopApp):
```

Comment:
Since the purpose of this is to test the "StopApp", I would suggest a rename to `StopAppTest`.

Reply:
Thanks for pointing this out, fixing it.